superset/templates/superset/base.html

```diff
@@ -17,10 +17,12 @@
   under the License.
 #}
 {% extends "appbuilder/baselayout.html" %}
+{% set app_favicon = appbuilder.app.config.get('APP_FAVICON', "/static/assets/images/favicon.png") %}
```

Comment:
Instead of setting the default value here, I think we should add APP_FAVICON and APP_FAVICON_TYPE to the config.py file and set it to this asset by default. Then you can also add comments documenting what the settings do